src/Generators/Webserver/Database/Drivers/PostgreSQL.php

```diff
@@ -143,8 +147,9 @@ protected function dropDatabase(IlluminateConnection $connection, array $config)
 
     protected function dropUser(IlluminateConnection $connection, array $config)
     {
-        if (config('tenancy.db.auto-delete-tenant-database-user') && $this->userExists($connection,
-                $config['username'])) {
+        if (config('tenancy.db.auto-delete-tenant-database-user')
+            && $this->userExists($connection, $config['username'])
+        ) {
```

Comment:
Can you return this to its original state?
I don't see any code changing except for some linting

Reply:
I had to make this change to get the checks to pass.